lib/plugin.js

```diff
@@ -139,8 +142,10 @@ const setupProgress = (axios, ctx) => {
     if (!currentRequests) {
       return
     }
-    const progress = ((e.loaded * 100) / (e.total * currentRequests))
-    $loading().set(Math.min(100, progress))
+    if (currentRequests > 1) {
```

Comment:
With this condition, we don't precisely show progress with single requests (but random jumps). Is it desired?